Week3/script.js

```diff
@@ -204,15 +203,15 @@ let newObject = {};
 
 let hackYourFuture = new Object();
 
-hackYourFuture.teachers = "Philipp, Rob, Unmesh, Bonan, Yash";
+hackYourFuture.teachers = ['Philipp', 'Rob', 'Unmesh', 'Bonan', 'Yash'];
 
 console.log(hackYourFuture);
 
 
 
         // 14
 
-hackYourFuture.langs = "HTML, CSS, CLI, GIT-Bash, JS";
+hackYourFuture.langs = ['HTML', 'CSS', 'CLI', 'GIT-Bash', 'JS'];
```

Comment:
Ah, i made a mistake in my first homework-review! I said that using an `array` is a better option and I think it is a better option. But now it's not clear which teacher has taught which class(es). Do you know how you can change these code so that it is clearer which teacher taught which classes(Hint: you can use object properties and arrays.

If you don't understand what I mean, please let me know! Then I will try to clarify more.